frontend/src/components/Leaflet/Leaflet.tsx

```diff
@@ -1,35 +1,39 @@
 import React, { useState } from 'react';
 import { Map, Marker, Popup, TileLayer } from 'react-leaflet';
-import MapDiv from './Leaflet.styling';
 import 'leaflet/dist/leaflet.css';
 
 const LeafletMap = () => {
-    // Set state for map coordinates
-    const [coordinates, setCoordinates] = useState({
-        lat: 51.505,
-        lng: -.09,
-        zoom: 13,
-    });
+  // Set state for map coordinates
+  const [coordinates, setCoordinates] = useState({
+    lat: 51.505,
+    lng: -0.09,
+    zoom: 13,
+  });

```

Comment:
Sweet deal here, I'm glad you found a solution for the mystery of the disappearing map 😁